src/languages/en.ts

```diff
@@ -2010,6 +2010,7 @@ export default {
                 reimbursedReportsDescription: 'Any time a report is paid using Expensify ACH, the corresponding bill payment will be created in the Xero account below.',
                 xeroBillPaymentAccount: 'Xero Bill Payment Account',
                 xeroInvoiceCollectionAccount: 'Xero Invoice Collections Account',
+                invoiceAccountSelectDescription: "As you've enabled exporting Invoices from Expensify to Xero, this is the account the Invoice will appear against once marked as Paid.",
```

Comment:
We can ask someone to verify the copy.

```suggestion
                invoiceAccountSelectDescription: "As you've enabled exporting invoices from Expensify to Xero, this is the account the invoice will appear against once marked as paid.",
```

Reply:
hmm I copied this from docs, let me check again

Reply:
I think current copy is correct

<img width="868" alt="Screenshot 2024-05-03 at 22 29 22" src="https://github.com/Expensify/App/assets/16502320/d1c20159-a388-4dad-8b0c-5e69ee8fd6ed">

Reply:
I agree it's correct when compared to the document, but at Expensify we generally follow sentence case. 

cc - @trjExpensify can chime in.

Reply:
Yeah, agreed with Manan!